CMakeLists.txt

```
@@ -17,6 +17,17 @@ include(ClangTidy)
 include(PedanticCompiler)
 
 find_package(reflection-cpp 0.4.0)
+find_package(Catch2 CONFIG REQUIRED)
+find_package(yaml-cpp CONFIG REQUIRED)
+
+option(LIGHTWEIGHT_CXX26REFLECTION "Enable experimental reflection support from c++26" OFF)
+
+if(LIGHTWEIGHT_CXX26REFLECTION)
```

```suggestion
if(LIGHTWEIGHT_CXX26_REFLECTION)
```
tbh, I think that would be way more readable.